### ISSUE TITLE
Typo in docs (structure_attr_fromtuple/dict)

### ISSUE THREAD

**Issue by MicaelJarniac** [python-attrs/cattrs#192]
* cattrs version: 1.9.0
* Python version: 3.9.7
* Operating System: Arch Linux (Manjaro)

> A global converter is provided for convenience as cattr.global_converter. The following functions implicitly use this global converter:
> * `cattr.structure`
> * `cattr.unstructure`
> * `cattr.structure_attr_fromtuple`
> * `cattr.structure_attr_fromdict`

https://cattrs.readthedocs.io/en/latest/converters.html#global-converter
https://github.com/python-attrs/cattrs/blame/main/docs/converters.rst#L19-L20

https://github.com/python-attrs/cattrs/blob/1b0d18ed3cbdcc1b34e949eaa30a407e36fa9978/src/cattr/converters.py#L414
https://github.com/python-attrs/cattrs/blob/1b0d18ed3cbdcc1b34e949eaa30a407e36fa9978/src/cattr/converters.py#L450

Basically, on the docs it says `structure_attr_fromtuple` and `structure_attr_fromdict`, but it should be `structure_attrs_fromtuple` and `structure_attrs_fromdict`.

I'll send a PR.
